_gpon/vendor.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: PON Vendor ID
+title: GPON Vendor ID
```

Comment:
```suggestion
title: ONU Vendor ID
```

Reply:
però anche gli OLT hanno i vendor ID....

Reply:
Si ma quel parametro è l'ONU Vendor ID, visto che parliamo in termini di attivazione